src/wrappers/themis/react-native-themis/src/index.d.ts

```diff
@@ -9,11 +11,10 @@ export declare function secureCellTokenProtectEncrypt64(symmetricKey64: String,
 export declare function secureCellTokenProtectDecrypt64(symmetricKey64: String, encrypted64: String, token64: String, context?: String): Promise<string>;
 export declare function secureCellContextImprintEncrypt64(symmetricKey64: String, plaintext: String, context: String): Promise<string>;
 export declare function secureCellContextImprintDecrypt64(symmetricKey64: String, encrypted64: String, context: String): Promise<string>;
-export declare function secureMessageSign64(plaintext: String, privateKey64: String, publicKey64: String): Promise<string>;
-export declare function secureMessageVerify64(signed64: String, privateKey64: String, publicKey64: String): Promise<string>;
+export declare function secureMessageSign64(plaintext: String, privateKey64: String, _publicKey64?: String): Promise<string>;
+export declare function secureMessageVerify64(signed64: String, _privateKey64: String | undefined, publicKey64: String): Promise<string>;
```

Comment:
Just curious, why does one get `?` but the other gets `| undefined`?

Reply:
I will add the ThemisTest app in the following pull request. It resolves the task. 
File d.ts is generated by Typescript. In the first case, we can omit the last argument in the sign method. It is entirely optional. In the second method, we indicate that the parameter is not gonna be used, but it should be placed in the middle even if it is undefined.

Reply:
No unit tests? 😞

![okay.jpg](https://user-images.githubusercontent.com/1256587/193809978-1759c86d-4de9-493c-9738-f05605214003.png)

Reply:
unfortunately, the fact that we can't write tests for the library with native modules. No sense to test only the JS part and it takes too much time. I made the app that uses 'unit tests' like technology. I will post it and you can run it on your local computer with a simulator or real device.

Reply:
It contains 70 tests.